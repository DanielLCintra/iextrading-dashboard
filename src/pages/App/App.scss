.App {
  width: 500px;
  max-width: 100%;
  margin: 0 auto;
  padding: 15px;
}
