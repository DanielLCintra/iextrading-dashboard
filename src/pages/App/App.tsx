import CssBaseline from '@material-ui/core/CssBaseline';
import * as React from 'react';
import { Route, Switch } from 'react-router-dom';
import Header from '../../components/Header';
import Home from '../Home';
import StockDetail from '../StockDetail';
import classes from './App.scss';

class App extends React.Component {
  public render() {
    return (
      <div className={classes.App}>
        <CssBaseline />
        <Header />
        <Switch>
          <Route exact={true} path="/" component={Home}/>
          <Route path="/stock/:symbol" component={StockDetail}/>
        </Switch>
      </div>
    );
  }
}

export default App;
