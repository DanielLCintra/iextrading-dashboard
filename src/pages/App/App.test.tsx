import { shallow } from 'enzyme';
import * as React from 'react';
import App from './App';

jest.mock('../../components/StockChart', () => () => null);

describe('App', () => {
  const wrapper = shallow(<App />);

  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
})
