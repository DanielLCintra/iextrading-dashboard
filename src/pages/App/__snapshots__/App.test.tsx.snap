// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div>
  <WithStyles(CssBaseline) />
  <Header />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/stock/:symbol"
    />
  </Switch>
</div>
`;
